django/conf/__init__.py

```diff
@@ -173,6 +178,8 @@ def __getattr__(self, name):
 
     def __setattr__(self, name, value):
         self._deleted.discard(name)
+        if name == 'DEFAULT_CONTENT_TYPE':
```

Comment:
Fair enough, makes sense though this doesn't seem to have been done in the past, e.g. b6ea059b4ab7a4ed7e84cad639df95fc9d61dd81 only put the warning in `Settings`